plasmapy/utils/checks.py

```diff
@@ -3,9 +3,11 @@
 
 import numpy as np
 from astropy import units as u
+from astropy.units import UnitsWarning
 from plasmapy.constants import c
 import warnings
 from plasmapy.utils.exceptions import RelativityWarning, RelativityError
+from textwrap import dedent
```

Comment:
Has this been added to dependencies or is it a built-in library?

Reply:
It's a built-in. I recall seeing heavy use of it in matplotlib.